src/test/java/com/github/cameltooling/lsp/internal/completion/CamelKubernetesServicesCompletionTest.java

```diff
@@ -193,6 +193,137 @@ void testKubernetesAutoCompletionPlaceholdersNotClosed() throws Exception {
 		assertThat(completions.get(1).getInsertText()).isEqualTo("}}something{{configmap:myMap/myKey2}}");
 	}
 
+	@Test
+	void testSimpleLanguage() throws Exception {
```

Comment:
I think the test for variables deserves a specific Test class as it is not related to Kubernetes only.

Reply:
You are right, I forgot about the name of the class